python/cuml/cuml/tests/test_metrics.py

```diff
@@ -15,6 +15,8 @@
 #
 
 import platform
+
```

Comment:
Likewise, the large diff here is just cleaning up our testing to both improve coverage and reuse tests across all the regression metrics. We have strictly better test coverage post this change.